docs/faq/troubleshooting/we-no-longer-have-access-to-this-repository.md

```diff
@@ -4,35 +4,33 @@ description: Troubleshoot and fix issues that prevent Codacy from cloning your p
 
 # We no longer have access to this repository, check your SSH keys
 
-Codacy uses SSH keys to clone your private repositories. Depending on the level of access that the user configuring the repository on Codacy has on the remote Git provider, an SSH key can be added either:
-
--   Directly to the repository itself, if the user has permissions to add SSH keys to the repository
--   To the user account, if the user only has read or commit permissions on the repository
-
-For this reason, some changes on your Git provider can prevent Codacy from cloning your private repository. When this happens, Codacy displays the error message "We no longer have access to this repository" on the Repository Dashboard page.
+Some changes on your Git provider can prevent Codacy from cloning your private repository. When this happens, Codacy displays the error message "We no longer have access to this repository" on the Repository Dashboard page.
```

Comment:
I suspect that "Some changes" is too generic and may leave the user scratching their head. I'm assuming that we're referring to the changes that we are trying to fix on this page.

I'm not sure this is the best way, though. Thoughts?

```suggestion
Some changes on your Git provider, like the ones described on this page, can prevent Codacy from cloning your private repository. When this happens, Codacy displays the error message "We no longer have access to this repository" on the Repository Dashboard page.
```

Reply:
This is a troubleshooting topic, so I believe we use "Some changes" because we also don't know the complete list of changes that can cause this error. 😄 

Your suggestion helps to link with the scenario below, but I'm not sure if we really need to be more explicit here. 🤔 
@prcr Any thoughts?

Reply:
This whole page (or error message, rather) is "messy" at this point. I'm perfectly comfortable with leaving further improvements out of the scope of the ongoing task and delegating them to [DOCS-345](https://codacy.atlassian.net/browse/DOCS-345).

Reply:
I added a comment to [DOCS-345](https://codacy.atlassian.net/browse/DOCS-345). 👍